pgmpy/tests/test_models/test_BayesianModel.py

```diff
@@ -855,9 +855,9 @@ def test_predict(self):
             ]
         )
 
-        np_test.assert_array_equal(p1.values.ravel(), p1_res)
```

Comment:
All the tests need to go to BayesianNetwork class. The BayesianModel class will be removed in future.